### PR TITLE
Feature/move input width to prototyping theme

### PR DESCRIPTION
**Pull request by patrickbenjamin1**
## What's new?

- move max-width on inputs into prototyping theme

## Ticket number(s) in JIRA (if internal)

ARM-XX

[board](https://rocketmakers.atlassian.net/jira/software/projects/HM/boards/172)

## Checklist

- [X] are your changes in Storybook?
- [X] are any breaking changes documented in `docs/migrating_from_oldstrong.md`?
- [X] does _everything_ have jsdoc?
- [X] is everything exported from index.ts?
- [X] are all new hooks added to `src/hooks/hooks.stories.mdx` or given their own docs in Storybook?
- [X] are all new SCSS mixins added to `src/theme/mixins.stories.mdx`?
- [X] are all new SCSS variables added to `src/theme/variables.stories.mdx`?

### CODE REVIEW

File: module/src/components/calendarInput/calendarInput.basic.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .arm-calendar-select {
   margin-right: 0;
   flex: 1;
+  min-width: 57px;
```
Comment (adamiprinciples) [inline module/src/components/calendarInput/calendarInput.basic.scss:4]: Magic number? If theres a reason for it should probably have a comment explaning why

File: module/src/components/listBox/listBox.basic.scss [hunk 2]
```diff
@@ -25,6 +20,7 @@
   display: flex;
   align-items: center;
   cursor: pointer;
+  min-width: 160px;
```
Comment (adamiprinciples) [inline module/src/components/listBox/listBox.basic.scss:23]: Does this specific value need to be documented? Should it come from a variable?

File: module/src/components/listBoxMulti/listBoxMulti.basic.scss [hunk 3]
```diff
@@ -22,6 +17,7 @@
   display: flex;
   align-items: center;
   margin-right: $spacing-large;
+  min-width: 160px;
```
Comment (adamiprinciples) [inline module/src/components/listBoxMulti/listBoxMulti.basic.scss:20]: Same as above

File: module/src/components/calendarInput/calendarInput.basic.scss (hunk 1 above)
Comment (patrickbenjamin1) [reply module/src/components/calendarInput/calendarInput.basic.scss]: it's been put in a variable and documented in storybook

File: module/src/components/listBox/listBox.basic.scss (hunk 2 above)
Comment (patrickbenjamin1) [reply module/src/components/listBox/listBox.basic.scss]: in a variable now

File: module/src/components/listBoxMulti/listBoxMulti.basic.scss (hunk 3 above)
Comment (patrickbenjamin1) [reply module/src/components/listBoxMulti/listBoxMulti.basic.scss]: in a variable now

**Review by adamiprinciples — APPROVED** [review]

**Pull request merged**